WordPress/Classes/Utility/CoreDataQuery.swift

```diff
@@ -0,0 +1,146 @@
+import Foundation
+
+struct CoreDataQuery<Result: NSManagedObject> {
```

Comment:
Nit: `Result` here is confusing to me because of the `Result<T, Error>` type. WDYT?